omas/tests/failed_imports.py

```diff
@@ -44,11 +46,15 @@
 except (ImportError, ServerSelectionTimeoutError) as _excp:
     failed_MONGO = _excp
 
+if hard_warnings:
```

Comment:
Best done with a `with` statement, starting at the beginning of this file

Reply:
Yes, good point